lighthouse-core/gather/gather-runner.js

```diff
@@ -491,6 +491,17 @@ class GatherRunner {
       const baseArtifacts = await GatherRunner.initializeBaseArtifacts(options);
       baseArtifacts.BenchmarkIndex = await getBenchmarkIndex(driver.executionContext);
 
+      if (options.settings.__internalMegaBenchmarkIndex) {
+        const bidxRunCount = options.settings.__internalMegaBenchmarkIndex;
+        // Keep the first one in there
+        const bidxes = [baseArtifacts.BenchmarkIndex];
+        for (let i = 0; i < bidxRunCount; i++) {
```

Comment:
It'd be simpler if the internal setting was just minus one'd here (And we asserted it either doesn't exist or is >= 1).

Reply:
as we're the only users i think this isn't a huge deal. and it'd complicate the impl